### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by TheHollidayInn**
This adds unit tests for:
- versions
- target
- use 
- update

### CODE REVIEW

File: kubos/test/test_target.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    def __init__(self):
+        self.set_target = None
+        self.target = None
+        self.list = None
+        self.plain = []
+
+class TargetTest(unittest.TestCase):
+    @patch('kubos.target.load_target_list')
+    @patch('kubos.target.get_project_type')
+    def test_add_options(self, get_project_type, load_target_list):
```
Comment (kyleparrott) [inline kubos/test/test_target.py]: At a high level, we don't really test the `add_options` function. This test becomes pretty brittle when we add new options or change existing ones (not that we do that a lot but it happens).  I know i missed this in #33.

File: kubos/test/test_update.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+from kubos import update
+from kubos.utils.constants import *
+
+class Args():
+    def __init__(self):
+        self.component = None
+
+class UpdateTest(unittest.TestCase):
+    def test_add_options(self):
```
Comment (kyleparrott) [inline kubos/test/test_update.py]: Same comment on not testing the  add options function. #34 has already changed the options added to the parser 😄

File: kubos/test/test_use.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+from kubos import use
+from kubos.utils.constants import *
+
+class Args():
+    def __init__(self):
+        self.set_version = None
+        self.branch = None
+
+class UseTest(unittest.TestCase):
+    def test_add_options(self):
```
Comment (kyleparrott) [inline kubos/test/test_use.py]: Same comment as above on `add_options`

File: kubos/test/test_versions.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+import unittest
+from mock import patch, MagicMock, call
+from pip.utils import get_installed_version
+
+from kubos import versions
+from kubos.utils.constants import *
+
+class VersionsTest(unittest.TestCase):
+    def test_add_options(self):
```
Comment (kyleparrott) [inline kubos/test/test_versions.py]: Same comment as above

File: kubos/test/test_target.py (hunk 1 above)
Comment (TheHollidayInn) [inline kubos/test/test_target.py]: Totally makes sense. These tests seem based on strings too much. I added it for completeness, but I will remove it. 👍

**Pull request merged**